web/client/epics/__tests__/featuregrid-test.js

```diff
@@ -1827,5 +1831,79 @@ describe('featuregrid Epics', () => {
             }
         }));
     });
+    describe('replayOnTimeDimensionChange', () => {
+        const SEARCH_URL = '/test-url';
+        const FILTER_OBJECT = { "dummy": "object" };
+        const TEST_STATE_OPEN_BASE = {
+            layers: {
+                flat: [{
+                    id: "TEST_LAYER",
+                    title: "Test Layer",
+                    name: 'editing:polygons',
+                    params: {
+                        viewParams: "a:b"
+                    }
+                }]
+            },
+            featuregrid: {
+                pagination: {
+                    size: 10
+                },
+                open: true,
+                selectedLayer: "TEST_LAYER",
+                changes: []
+            }
+        };
+        const isSameQuery = a => {
+            expect(a.type).toBe(QUERY_CREATE);
+            expect(a.searchUrl).toBe(SEARCH_URL);
+            expect(a.filterObj).toBe(FILTER_OBJECT);
+        };
+        it('toggle on setTimeSync if FG is open', done => {
+            testEpic(replayOnTimeDimensionChange, 1, [query(SEARCH_URL, FILTER_OBJECT), setTimeSync(false)], ([a]) => {
+                isSameQuery(a);
+                done();
+            }, TEST_STATE_OPEN_BASE);
+        });
+        const TEST_STATE_CLOSED = set('featuregrid.open', false, TEST_STATE_OPEN_BASE);
```

Comment:
I would move this constants INSIDE the test

Reply:
only  TEST_STATE_TIME_ACTIVE_CLOSED is private, the other ones are shared, I'll move them on top with the other constants